### PR TITLE
material-ui updated to v4.8.3+

### PR DESCRIPTION
**Pull request by theprover97**
#2241 
material-ui/lab has been also updated to "4.0.0-alpha.39" which introduces the Alert component.

### CODE REVIEW

**Comment by theprover97** [comment]
Hey @helfi92 , I read the updated files in Material-UI. The colors are same but those rendered for the ' label text ' and ' underline ' are matching with the background after onClick event. I tried using styles with the TextField in SignInDialog, but it doesn't work for different shades. Could you help me out in this?

File: ui/package.json [hunk 1]
```diff
@@ -17,9 +17,9 @@
     "create:fragment-matcher": "dotenv node src/scripts/queryServer"
   },
   "dependencies": {
-    "@material-ui/core": "^4.5.1",
+    "@material-ui/core": "^4.8.3",
     "@material-ui/icons": "^4.5.1",
-    "@material-ui/lab": "4.0.0-alpha.33",
+    "@material-ui/lab": "4.0.0-alpha.39",
```
Comment (helfi92) [inline ui/package.json]: Let's avoid changing other libraries to minimize the chance of something else breaking.

**Comment by aryamanpuri** [comment]
@theprover97 Are you working on it, or should I finish this up?
@helfi92

**Comment by theprover97** [comment]
@aryamanpuri , I am still working on it. I will push the changes within 4 days.

**Comment by helfi92** [comment]
@theprover97 How is it going?

**Comment by theprover97** [comment]
I am sorry I can not continue work on this. Others are welcome to finish this up.

**Comment by helfi92** [comment]
No worries. Thanks for letting us know!

**Comment by helfi92** [comment]
@aryamanpuri Still interested in finishing this up?

**Comment by aryamanpuri** [comment]
> @aryamanpuri Still interested in finishing this up?

I will start working on this from monday onwards.

**Pull request closed without merging**